__tests__/components/Videos/Upload/index.test.tsx

```
@@ -0,0 +1,367 @@
+
+describe('HookMultiStepForm', () => {
+  describe('Token Gating', () => {
+    const mockRouterPush = vi.fn();
+    
+    beforeEach(() => {
+      vi.clearAllMocks();
+      
+      // Mock the router push method
+      vi.mocked(useRouter).mockReturnValue({
+        push: mockRouterPush,
+      } as any);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)